src/script/microbit-interfacing/MicrobitSerial.ts

```diff
@@ -80,6 +94,19 @@ export class MicrobitSerial implements MicrobitConnection {
       await this.handshake();
       stateOnConnected(DeviceRequestStates.INPUT);
 
+      // Check for connection lost
+      if (this.connectionCheckIntervalId === undefined) {
+        this.connectionCheckIntervalId = setInterval(async () => {
+          const allowedTimeWithoutMessageInMs = isDevMode ? 2000 : 10000;
```

Comment:
We can use StaticConfiguration.connectTimeoutDuration for the 10 second timeout to match Bluetooth.

Reply:
Nice, didn't know that existed